backend/src/Service/Order/OrderService.php

```diff
@@ -13,11 +13,23 @@
 use App\Constant\Subscription\SubscriptionConstant;
 use App\Service\Subscription\SubscriptionService;
 use App\Service\Notification\NotificationLocalService;
+use Symfony\Component\DependencyInjection\ParameterBag\ParameterBagInterface;
 
 class OrderService
 {
-    public function __construct(private AutoMapping $autoMapping, private OrderManager $orderManager, private SubscriptionService $subscriptionService, private NotificationLocalService $notificationLocalService)
+    private AutoMapping $autoMapping;
+    private OrderManager $orderManager;
+    private SubscriptionService $subscriptionService;
+    private NotificationLocalService $notificationLocalService;
+    private string $params;
+
+    public function __construct(AutoMapping $autoMapping, OrderManager $orderManager, SubscriptionService $subscriptionService, NotificationLocalService $notificationLocalService, ParameterBagInterface $params)
     {
+       $this->params = $params->get('upload_base_url') . '/';
```

Comment:
We have to use UploadFileHelperService to get ypload base url instead of this way.